cmd/runitor/main.go

```diff
@@ -71,7 +71,7 @@ func main() {
 	}
 
 	if *apiURL == internal.DefaultBaseURL {
-		v, ok := os.LookupEnv("CHECK_API_URL")
+		v, ok := os.LookupEnv("HC_API_URL")
```

Comment:
Here's an unlikely yet still possible case:
- HC_API_URL=https://example.com/hc
- Command: `runitor -uuid foo-bar-baz -api-url https://hc-ping.com`

The intention of the user is to hit hc-ping.com but the environment variable is set. The promise of command line argument getting precedence over the env var is not kept. So it's different than `uuid` because there's no natural default value for it.

I may recommend using `flag.Visit` in here.

```
urlFromArgs := false
flag.Visit(func(f *flag.Flag)) {
  if f.Name == "api-url" {
    urlFromArgs = true
  }
}
```